runners/spark/job-server/build.gradle

```diff
@@ -73,6 +73,8 @@ runShadow {
     args += ["--clean-artifacts-per-job=${project.property('cleanArtifactsPerJob')}"]
   if (project.hasProperty('sparkMasterUrl'))
     args += ["--spark-master-url=${project.property('sparkMasterUrl')}"]
+  if (project.hasProperty('sparkHistoryDir'))
```

Comment:
I don't think we should add `sparkHistoryDir` as an argument to the job server.

(I don't think sparkMasterUrl is really necessary either, since it was already a pipeline option. But it's probably too late to remove it now.)

Reply:
Pipeline options are coded across all SDKs, therefore two extra options require to change pipeline options in Python SDK, Java SDK, Go SDK. My proposition is to keep it that way, and change every SDKs pipeline options gradually in other smaller PRs. Could we do it in such a way or do you insist on doing all these changes here in this PR?